Bencodex/Types/List.cs

```diff
@@ -50,8 +53,12 @@ public string Inspection
             }
         }
 
+        int IReadOnlyCollection<IValue>.Count => Value.Length;
```

Comment:
Haven't the `Bencodex.Types.List` type implemented the `IReadOnlyCollection<IValue>` interface?

- If it haven't implemented the interface how can this method be added without implementing the interface?
- If it have implemented isn't this method redundant?  (Is the interface implementation already satisfied without this?)

Reply:
Actually, it was my mistake from rebasing. The lines were updated at #25 and not related to this pull-request. So I amended it.